src/style.css

```diff
@@ -0,0 +1,147 @@
+* {
+  margin: 0;
+}
+
+.photo {
+  display: flex;
+  height: 134px;
+  width: 160px;
+  margin-top: 32px;
+  margin-bottom: 40px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  align-items: center;
+  width: 200px;
+  padding: 0 16px;
+}
+
+.card__title {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  width: 166px;
+  height: 36px;
+  box-sizing: border-box;
+  color: #060b35;
+}
+
+.card__code {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  align-self: start;
+  margin-top: 4px;
+}
+
+.rating {
+  display: flex;
+  font-family: "Roboto", sans-serif;
```

Comment:
add font-family to html or body, no need to add it to every element, it's inherited property